File: Algorithms/1550/1550.c
int threeConsecutiveOdds(int* arr, int arrSize) {
    int count = 0;  // Counter to track consecutive odd numbers

    // Loop through each element in the array
    for(int i = 0; i < arrSize; i++){
        if ((arr[i] % 2) == 1){ 
            // If current element is odd, increment counter
            count += 1; 
        }
        else{
            // If it's even, reset the counter
            count = 0;
        }

        // If we've seen three consecutive odds, return true (1)
        if (count == 3){
            return 1;
        }
    }

    // No three consecutive odd numbers found
    return 0; 
}

File: Algorithms/1550/1550.py
class Solution:
    def threeConsecutiveOdds(self, arr) -> bool:
        # Counter to track consecutive odd numbers
        out = 0 
        
        # Loop through each number in the array
        for i in arr:
            # Check if the number is odd
            if i % 2 != 0:
                out += 1  # Increment counter if odd
            else:
                out = 0  # Reset counter if even

            # If we've seen 3 odds in a row, return True
            if out == 3:
                return True 
        
        # If loop finishes without finding 3 consecutive odds
        return False
